Reflections/Week_13_Reflection.md

```diff
@@ -1,22 +1,22 @@
 1. What have you accomplished this week? (Please list each accomplishment, and explain) 
-  - I was able to conncect the Linear Actuator to the currently built system. (D4.5 shows this.)
-  - I was able to get it to work together with only a few hickups, since they were built sepertly, so new parts needed to be intergrated or moved around. (D4.5 shows this.)
+  - I was able to connect the Linear Actuator to the currently built system. (D4.5 shows this.)
```

Comment:
It's great that you connected everything together. Does the remote control app work the same way?